main.cpp

```diff
@@ -27,12 +32,14 @@ int main()
         cv::Mat image;
         bool ret;
 
-        detect laneDetection(510, 280, camera);
+        detect laneDetection(FRAME_WIDTH, FRAME_HEIGHT, camera);
+        laneKeeping lane_keeping_obj(FRAME_WIDTH, FRAME_HEIGHT, camera);
```

Comment:
`ret = cap.read(image);` to read the first frame of the video and then : 
```
detect laneDetection(image.cols, image.rows, camera);
laneKeeping lane_keeping_obj(image.cols, image.rows, camera); 
```

Reply:
Done ✅
```c++
bool ret = cap.read(image);
detect laneDetection(image.cols, image.rows, camera);
laneKeeping laneKeeping(image.cols, image.rows, camera);
```